docs/en/userguide/index.html

```diff
@@ -1,8 +1,8 @@
-<p>This page documents how to use the Document Converter module.</p>
+<h2 id="module">Document Converter</h2>
+<p>The Document Converter module enables Users to import a tree structure of pages, from Sections and Headings in a Word Document. It is ideal for when you want to quickly populate your website from your existing document resources.</p>
 <h2 id="usage">Usage</h2>
 <p>Whether the document converter options will appear on your website it depends on how it has been set up. For this manual we will assume the conversion has been allowed globally.</p>
-<p>First of all it is important to know that the import process will replace the content of the page your are importing to, and it also has a licence to remove and replace all its children. It will still be possible to get the content back from the version history of the CMS, but it's good to keep that in mind.</p>
-<h3 id="interface">Interface</h3>
+<p>Please note that the import process will replace the content of the page your are importing to, and it also has a licence to remove and replace all its children. It will still be possible to get the content back from the version history of the CMS, but it's good to keep that in mind.</p>
```

Comment:
Hmmm... why do we have an html file here too? Can you delete it and just use the markdown one please?

Reply:
Yeah, I wasn't sure either. Thought I'd keep it in case but will do.